exaudfclient/base/script_options_parser/ctpg/script_option_lines_ctpg.cc

```diff
@@ -52,19 +52,36 @@ auto&& add_option(Option&& e, options_type&& ob)
     return std::move(ob);
 }
 
+const auto convert_escape_seq(std::string_view escape_seq) {
+    std::string retVal;
+    if (escape_seq == R"_(\;)_") {
+        retVal = ";";
+    } else if (escape_seq == R"_(\n)_") {
+        retVal = "\n";
+    } else if (escape_seq == R"_(\r)_") {
+        retVal = "\r";
+    } else {
+        throw std::runtime_error(std::string("Internal parser error: Unexpected escape sequence " + std::string(escape_seq)));
+    }
+
+    return retVal;
+}
+
 
 
 constexpr char alpha_numeric_pattern[] = R"_([0-9a-zA-Z_]+)_";
 constexpr char not_semicolon_pattern[] = R"_([^;])_";
 constexpr char whitespaces_pattern[] = R"_([ \x09\x0c\x0b]+)_";
+constexpr char escape_pattern[] = R"_(\\;|\\n|\\r)_";
```

Comment:
We still missing whitespace at the beginning of the value.The question is how we escape that. Such that we can specify a value like this "  MyValue". Possible ideas the first whitespace needs to be "\ " or it could be its hex sequence. And, it also should work with tabs.

Reply:
Done